pages/common/compgen.md

```diff
@@ -7,6 +7,10 @@
 
 `compgen -c`
 
+- List all commands that you could run that start with `str`:
+
+`compgen -c {{str}}`
```

Comment:
```suggestion
`compgen -c str`
```

Reply:
I'm going to change the description instead. In my opinion it's more important to keep the interchangeable parts of a command as placeholders.